packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -2672,7 +2672,10 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
     if (!_shouldCreateInputConnection) {
       _closeInputConnectionIfNeeded();
     } else if (oldWidget.readOnly && _hasFocus) {
-      _openInputConnection();
+      // We have to wait because of #126324
```

Comment:
```
// _openInputConnection must be called after layout information is available.
// See https://github.com/flutter/flutter/issues/126312
```

Reply:
Then I would add a comment on the definition of _openInputConnection and/or _updateSizeAndTransform:

```
// Must be called after layout.
```

Reply:
done